examples/Planning/GetAdGroupCriterionCpcBidSimulations.php

```diff
@@ -34,10 +34,10 @@
 use Google\ApiCore\ApiException;
 
 /**
- * This example gets all available keyword bid simulations for a given ad group.
+ * This example gets all available CPC bid simulations for a given ad group.
```

Comment:
CPC along wouldn't represent the "keyword", as CPC denotes cost-per-click. I think we should include "ad group criterion" as well.
Can you include ad group criterion (like you do in the file name) all over the places?

Reply:
SG